scripts/18-convert2xd.py

```diff
@@ -33,6 +33,8 @@ def main():
     global args
     parsers = {
         '.xml': [parse_ccxml, parse_uxml],
+        '.xml.1': [parse_ccxml, parse_uxml],
```

Comment:
Were some on universal/ - fcx2007-01-02.xml.2
Maybe it's a good idea to skip such...